src/gui/guiEditBoxWithScrollbar.cpp

```diff
@@ -57,6 +57,7 @@ GUIEditBoxWithScrollBar::GUIEditBoxWithScrollBar(const wchar_t* text, bool borde
 
 	if (has_vscrollbar) {
 		createVScrollBar();
+		RelativeRect.LowerRightCorner.X -= m_scrollbar_width + 4;
```

Comment:
Could you please move those lines into `createVScrollBar()`? Functionally it's the same but in case someone wants to use the scrollbars for a different feature, they will again run into the very same issue if it's not done in the scrollbar constructor function.